libs/geography/src/reducers/country-entities/country-entities.reducer.spec.ts

```diff
@@ -54,13 +58,35 @@ describe('Geography | Reducer | CountryEntities', () => {
     let result;
 
     beforeEach(() => {
-      const countryListSuccess = new DaffCountryListSuccess([country]);
+      const countryListSuccess = new DaffCountryListSuccess([mockCountry]);
 
       result = reducer(initialState, countryListSuccess);
     });
 
     it('should set countries from action.payload', () => {
-      expect(result.entities).toEqual({[countryId]: jasmine.objectContaining(country)})
+      expect(result.entities).toEqual({[countryId]: jasmine.objectContaining(mockCountry)})
+    });
+
+    describe('when a country has previously been fully loaded', () => {
```

Comment:
I think you should add one more test for the case where the list request gives subdivisions

Reply:
done